src/prpy/planning/vectorfield.py

```diff
@@ -357,7 +357,7 @@ def fn_callback(t, q):
         output_path.Insert(0, path.GetWaypoints(0, cached_index), cspec)
 
         # Add a segment for the feasible part of the last integration step.
-        output_path.Insert(output_path.GetNumWaypoints() - 1,
+        output_path.Insert(output_path.GetNumWaypoints(),
```

Comment:
Is this correct? Or is there still an off-by-one-error?

I always get confused about the `Insert` method on `Trajectory`s.